docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: model
+title: Indonesion NER Model
+author: John Snow Labs
+name: xlm_roberta_large_token_classification_ner
+date: 2021-12-03
+tags: [id, indonesian, xlm, roberta, xlm_roberta, ner, open_source]
+task: Named Entity Recognition
+language: id
+edition: Spark NLP 3.3.3
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is imported from `Hugging Face-models`. It is the Indonesian fine-tuned of "xlm-roberta-large" language model.
```

Comment:
This model has been imported from `Hugging Face`. It's been trained using a `xlm-roberta-large` finetuned model on Indonesian texts.